src/main/solr/dssolr/conf/schema.xml

```diff
@@ -582,6 +582,11 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?example     www.example.com/streaming/bart-access-copies-tv/00/00/00/000000e1-0000-462a-a2b4-7488244fcca7/playlist.m3u8"?>
   </field>
 
+  <field name="date_modified" type="pdate" stored="true">
```

Comment:
This will be changed to something like `internal_mTime` or something like that, right?

Reply:
Yes, will change to probably something like internal_storage_mTime to keep the  internal_ prefix and indicate, that the value originates from our internal ds-storage and not say preservica. This value will also be included in the internal map of the schema.org JSON, while the dateModified there should be reserved for a value related to metadata modification extracted from origin